tools/detekt/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ plugins {
 
 dependencies {
     implementation(Dependencies.Libraries.Kotlin)
+    implementation(Dependencies.Libraries.KotlinReflect)
```

Comment:
why it is needed here?

Reply:
because detekt uses an old kotlin reflect version under the hood and the compiler complains and recommends to provide the new kotlin-reflect version into the deps.